test/snoowrap.spec.js

```diff
@@ -1183,6 +1183,40 @@ describe('snoowrap', function () {
     });
   });
 
+  describe('fetching subreddits by query', () => {
+    it('should fail because query length is over 50 characters', async () => {
+      const query = 'learnprogramminglearnprogramminglearnprogramminglearnprogramminglearnprogramminglearnprogramminglearnprogramming';
```

Comment:
Would be more readable to do 
```suggestion
      // Get a query that's longer than 50 characters
      const query = "x".repeat(51);
```